docs/apache-airflow-providers-amazon/auth-manager/setup/identity-center.rst

```diff
@@ -0,0 +1,147 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+=================================
+Configure AWS IAM Identity Center
+=================================
+
+In order to use the AWS auth manager, you first need to configure `AWS IAM Identity Center <https://aws.amazon.com/iam/identity-center/>`_.
+AWS IAM Identity Center is used by the AWS auth manager for authentication purposes (login and logout).
+The Airflow environment administrator then uses Identity Center to manage users and groups.
+
+Create resources
+================
+
+The AWS auth manager needs two resources in AWS IAM Identity Center: an instance and an application.
+You can create them either through the vended CLI command or manually.
+
+Create resources with CLI
+-------------------------
+
+.. note::
+  The CLI command is not compatible with AWS accounts that are managed through AWS organizations.
+  If your AWS account is managed through an AWS organization, please follow the
+  :ref:`manual configuration <identity_center_manual_configuration>`.
+
+.. note::
+  In order to create all resources needed by the AWS auth manager, you can use the CLI command vended as part of the AWS auth manager.
+  In order to use it, you first need to set the AWS auth manager as auth manager in Airflow config.
```

Comment:
To create all necessary resources for the AWS Auth Manager, you can utilize the CLI command provided as part of the AWS auth manager. Before executing the command, ensure the AWS auth manager is configured as the auth manager for the Airflow instance.